src/Plot/Scale.elm

```diff
@@ -0,0 +1,32 @@
+module Plot.Scale where
+
+import Private.Scale.Linear as LinearScale
+import Private.Scale.OrdinalBands as OrdinalBands
+import Private.Extras.Set as Set exposing (Set)
+import Private.Scale exposing (Scale)
+
+type alias LinearScale = Scale Set Float
+type alias OrdinalScale = Scale (List String) String
+
+linear : (Float, Float) -> (Float, Float) -> Int -> LinearScale
+linear domain range numTicks =
+  { domain = Set.createFromTuple domain
+  , range = Set.createFromTuple range
+  , interpolate = LinearScale.interpolate
+  , uninterpolate = LinearScale.uninterpolate
+  , createTicks = LinearScale.createTicks numTicks
+  , inDomain = LinearScale.inDomain
+  }
+
+ordinalBands : List String -> (Float, Float) -> Float -> Float -> OrdinalScale
+ordinalBands domain range padding outerPadding =
```

Comment:
This is a place holder for me to investigate further. I may delete this comment. Padding & outerPadding work here, but ... sort of convolute the subject? creating ticks is usually a purposeful choice, especially when you're not intending to render an axis.